src/expensive_services_detail/cost_metrics_of_expensive_services.py

```diff
@@ -173,7 +173,7 @@ def lambda_handler(event, context):
         for resource in top_5_resources:
             resourcedata = {
                 "Account": account_detail,
-                "Region": f"{region} ({region_names.get(region, 'unknown region name')})",
+                "Region": f"{region}-{region_names.get(region, 'unknown region name')}",
```

Comment:
@muhammad-irfan-xgrid Seeing the code, looks like issue is resolved, but can you also paste a screenshot from the dashboard for the verification?

Reply:
Here it is. 
![image](https://github.com/XgridInc/xc3/assets/145985682/78c2806f-9547-4165-a420-cd309251b96a)